__tests__/commands/install/integration.js

```diff
@@ -208,13 +208,17 @@ test.concurrent('hoisting should factor ignored dependencies', async () => {
 });
 
 test.concurrent('--production flag ignores dev dependencies', () => {
-  return runInstall({production: true}, 'install-production', async (config) => {
+  return runInstall({production: true}, 'install-production-without-dev', async (config) => {
```

Comment:
I think it's better to add a new test rather than change existing one if the new one covers a different case